src/manifest/core.clj

```diff
@@ -1,15 +1,29 @@
 (ns manifest.core
   (:require [clojure.java.io :as io]
+            [clojure.string]
             [clojure.walk :as walk])
   (:import [java.util.jar Manifest]
            [java.net URL]))
 
 ;; Note that the manifest you want will only be on the classpath if it lives in the uberjar
 
+(defn- join-continuations [input]
+  (let [input (vec input)
+        continuation-indexes (keep-indexed #(when (= \space (first %2)) %1) input)
+        spliced (reduce (fn [v i]
+                          (update-in v [(dec i)] str
+                                     (clojure.string/replace-first (get v i) " " "")))
+                        input
+                        (reverse continuation-indexes))
+        continuation-indexes (set continuation-indexes)
+        spliced (keep-indexed (fn [i v] (when-not (continuation-indexes i) v)) spliced)]
+    spliced))
```

Comment:
Well spotted - removed the last `splice` shadowing.

The shadowing of `continuation-indexes` can be removed if a reverse sorted-set is used, i.e. updating with something like the following:

`(sorted-set-by > (keep-indexed #(when (= \space (first %2)) %1) input))`

Would that be preferable?